### ISSUE TITLE
The blueprint files don't seem to work

### ISSUE THREAD

**Issue by tgsparky** [dwainscheeren/dwains-dashboard-blueprints#132]
Hi
I'm loving your dash board so far. I am having an issue with all of the Github blueprints. I paste them in and I press the install blueprint button and it does nothing. Almost like I havent pasted anything in. I'm not sure if it's me or not.

**Comment by Ralphy140** [comment]
Same issue for me

**Comment by Fendtie** [comment]
Same here

**Comment by lisap516** [comment]
same issue here too

**Comment by cmdshft** [comment]
Same issue

**Comment by southwestflavorz** [comment]
I"m also having the same issue. Can't install blueprints.

**Comment by joea27** [comment]
Everytime I try this theme theres always something that doesnt work. I'm over it. Uninstall

**Comment by Ralphy140** [comment]
> Everytime I try this theme theres always something that doesnt work. I'm over it. Uninstall

You know this is FREE software built by someone who has volunteered their time. You might not like the support on free software(none) but you dont need to be rude! This is how you make people stop wanting to help other people. And yes i have had stuff not work but most of the time it gets resolved but when someone pulls their work because of people who insult them for giving something away for free it impact everyone else who is able to use it without issue. So perhaps next time try be a little more polite! /Rant

**Comment by Bosty01** [comment]
https://github.com/dwainscheeren/dwains-lovelace-dashboard/issues/822

This is the answer. Make sure you restart HA.

**Comment by hasancst** [comment]
Thank you for the beautiful work, i just recently trying using HA, and found dwains theme i try to use blueprint but also got an issue not working.

**Comment by Ralphy140** [comment]
> Thank you for the beautiful work, i just recently trying using HA, and found dwains theme i try to use blueprint but also got an issue not working.

Can confirm that the fix linked above does work. https://github.com/dwainscheeren/dwains-dashboard-blueprints/issues/132#issuecomment-2253522015 you will need to modify a file and then restart HA(not just reload the YAML) it then lets you import the blueprints.

**Comment by hasancst** [comment]
@Ralphy140 yes, it's working thank you. but first time i use that solution my dwains dashboard gone but next i restart it again then readd the code for the second time and restart my ha it's back normally
